frontend/src/Socket.js

```diff
@@ -2,9 +2,9 @@ import socketIOClient from 'socket.io-client';
 
 /* eslint-disable new-cap */
 // Uncomment if you would like to use the deployed server
-const socket = new socketIOClient('mafia-hands-backend-cd.herokuapp.com');
+// const socket = new socketIOClient('mafia-hands-backend-cd.herokuapp.com');
 
 // Uncomment if you would like to use a local server to test server changes
-// const socket = new socketIOClient('localhost:4001');
```

Comment:
I'm not really reviewing this but just spotted this. a2-main does not use the local server. Should this change be in a separate basic issue. It does not fit so much with this issue, and impacts all other PR's.

Reply:
Yeah, we had a discussion in the team 1 channel about this. We decided we should update it as we have to manually change to the dev server every time. I guess we could make a seperate issue for it.